src/javascript/extendable_audio.js

```diff
@@ -0,0 +1,46 @@
+/// <reference path="./extendable_audio.d.ts" />
+
+const SKIP_PROPERTIES = ['dispatchEvent', 'addEventListener', 'removeEventListener'];
```

Comment:
These three methods are defined on the audio element and not defined by the class yet, but we still need to not copy them over for ... reasons.